### DISCUSSION TITLE
Non-technical CIPs

### DISCUSSION THREAD

**Discussion by joecartesi** [cartesi/cips#36] [Ideas]
Hi, please provide any feedback you may have for this proposal: https://github.com/cartesi/cips/issues/34

Thank you!

**Comment by packetstracer** [comment]
Looks good to us. This helps with community driven evolution.

As proposals are non-technical probably the proposers won't have a technical profile, so may be good to lower if possible the technical "barriers" they may encounter in the CIP process. So, the easier the better, specially with non tech profiles.
